src/dataflow-types/src/client/controller.rs

```diff
@@ -131,3 +128,15 @@ impl<C> Controller<C> {
         }
     }
 }
+
+/// Compaction policies for collections maintained by `Controller`.
+pub enum ReadPolicy<T = mz_repr::Timestamp> {
+    /// Maintain the collection as valid from this frontier onward.
+    ValidFrom(Antichain<T>),
+    /// Maintain the collection as valid from a function of the write frontier.
+    ///
+    /// This function will only be re-evaluated when the write frontier changes.
+    /// If the intended behavior is to change in response to external signals,
+    /// consider using the `StopAt` variant to manually pilot compaction.
```

Comment:
Did `StopAt` get renamed to `ValidFrom`?

Reply:
Oh dear. It did.